src/actors/ActorId.ts

```diff
@@ -11,7 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { v4 as uuidv4 } from "uuid";
+import * as crypto from "crypto";
```

Comment:
Should we do?
```suggestion
import { crypto } from "crypto";
```

Reply:
It should be `import { randomUUID } from "crypto";` then, adapted accordingly :)